.github/pull_request_template.md

```diff
@@ -2,3 +2,4 @@
 ### Done checklist
 - [ ] docs/SPEC.md updated
 - [ ] Test cases written
+- [ ] Sign [CLA](https://www.influxdata.com/legal/cla/) (if not already signed)
```

Comment:
Do we need this anymore or can we modify the classee checker to post a comment?

Reply:
I would love to get it to post a comment actually. It doesn't do that yet sadly. I will look into adding that.